### ISSUE TITLE
zbar_ros fails to build in melodic

### ISSUE THREAD

**Issue by argenos** [b-it-bots/zbar_ros#1]
The following error appears:

```
/melodic/src/zbar_ros/src/barcode_reader_nodelet.cpp:136:24: error: expected constructor, destructor, or type conversion before ‘(’ token
 PLUGINLIB_DECLARE_CLASS(zbar_ros, BarcodeReaderNodelet, zbar_ros::BarcodeReaderNodelet, nodelet::Nodelet);
                        ^
CMakeFiles/barcode_reader_nodelet.dir/build.make:62: recipe for target 'CMakeFiles/barcode_reader_nodelet.dir/src/barcode_reader_nodelet.cpp.o' failed
make[2]: *** [CMakeFiles/barcode_reader_nodelet.dir/src/barcode_reader_nodelet.cpp.o] Error 1
CMakeFiles/Makefile2:131: recipe for target 'CMakeFiles/barcode_reader_nodelet.dir/all' failed
make[1]: *** [CMakeFiles/barcode_reader_nodelet.dir/all] Error 2
Makefile:140: recipe for target 'all' failed
make: *** [all] Error 2

```

**Comment by minhnh** [comment]
resolved by e731c6a ?

**Comment by argenos** [comment]
Yes!
